test/jasmine/tests/lib_test.js

```diff
@@ -1558,12 +1558,16 @@ describe('Test lib.js:', function() {
 
     describe('cleanNumber', function() {
         it('should return finite numbers untouched', function() {
-            [
-                0, 1, 2, 1234.567,
-                -1, -100, -999.999,
-                Number.MAX_VALUE, Number.MIN_VALUE, Number.EPSILON,
-                -Number.MAX_VALUE, -Number.MIN_VALUE, -Number.EPSILON
-            ].forEach(function(v) {
+            var vals = [0, 1, 2, 1234.567, -1, -100, -999.999];
+
+            if(!Lib.isIE()) {
+                vals.push(
+                    Number.MAX_VALUE, Number.MIN_VALUE, Number.EPSILON,
+                    -Number.MAX_VALUE, -Number.MIN_VALUE, -Number.EPSILON
+                );
```

Comment:
Oh dear, IE doesn't support these? We use `MAX_VALUE`:

https://github.com/plotly/plotly.js/blob/077a1d770cb7dc9fa8c2588bfb64a80607cf9095/src/constants/numerical.js#L26

Reply:
fixed in -> https://github.com/plotly/plotly.js/pull/3285/commits/50218deb708cab59260df5bf5096d5a61988eb2d